resources/views/auth/application/application.blade.php

```diff
@@ -11,11 +11,64 @@
                     @endif
                 </div>
                 <div class="col s12 xl8">
-{{--                    @can('editStatus', \App\Models\Application::class)--}}
-{{--                        @livewire('application-status-update', ['application' => $user->application])--}}
-{{--                    @endcan--}}
 
-                    <div class="card-title">{{ $user->name }}</div>
+                    <div class="right" style="margin:5px;width:150px">
+                        @can('viewUnfinished', \App\Models\Application::class)
+                            @if(!$user->application->submitted)
+                                <span class="new badge scale-transition red tag" style="display: block;"
+                                      data-badge-caption="">
+                                Nincs véglegesítve
+                                </span>
+                            @endif
```

Comment:
For me, it is a bit too close to the other badges; maybe we should give a larger margin to it, or an additional divider. But this is probably a question of taste.

Reply:
I made it uppercase